README.md

```diff
@@ -24,6 +25,36 @@ npm run build
 npm run test
 ```
 
+## Live testnet testing
+
+1. Ensure that the contract deployments repository supports the testnet you will be using. Supported chains can be found [here](https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/info/testnet.json).
```

Comment:
you also need to update hardhat config in this repo to point to your local modified info file

Reply:
I'll add instructions for this